tests/FakeItEasy.Tests.TestHelpers.FSharp/FakeItEasy.Tests.TestHelpers.FSharp.fsproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(TestHelpersTargetFrameworks)</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Remove="xunit.runner.visualstudio" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="*.fs"/>

```

Comment:
Isn't it implicit like in C#?

Reply:
I needed to add the `Compile`. I can explode the \*.

Reply:
Yeah, it would probably be safer. Or maybe just let Visual Studio do its own thing?

Reply:
What's "its own thing"? Have me add the file through the GUI? I can try it.

I'm back. I tried it. It did the same thing I did "by hand".

Reply:
Oh? Well, in this case, don't bother changing anything! I can merge as soon as you remove the [WIP] from the title